### PR TITLE
Reverts additional move effect macros

### PR DESCRIPTION
**Pull request by AlexOn1ine**
Reverted macros from PR #4096 

I think those specific macros made the code hard to read and we shouldn't make a simple loop more complicated then it should even if it's duplicating code.

### CODE REVIEW

File: include/pokemon.h [hunk 1]
```diff
@@ -467,7 +467,8 @@ struct MoveInfo
     u32 strikeCount:4; // Max 15 hits. Defaults to 1 if not set. May apply its effect on each hit.
     u32 criticalHitStage:2;
     u32 alwaysCriticalHit:1;
-    // 14 bits left to complete this word - continues into flags
+    u32 numAdditionalEffects:2; // limited to 3 - don't want to get too crazy
```
Comment (AsparagusEduardo) [inline include/pokemon.h:470]: From what I recall, the previous instance let us stack up to 15 secondary effects.
Can we preserve this, or would you prefer it to be done in a follow up PR?

Comment (AlexOn1ine) [reply include/pokemon.h:470]: I would rather prefer if someone else then me does it (CF?)

Comment (AsparagusEduardo) [reply include/pokemon.h:470]: gotcha

**Review by AsparagusEduardo — APPROVED** [review]

File: include/pokemon.h (hunk 1 above)
Comment (cfmnephrite) [reply include/pokemon.h:470]: I wrote 15 in the move guide so that should have been 4 bits. It's not like there was no room for it.

**Pull request merged**